packages/mds-agency/server.ts

```diff
@@ -15,6 +15,10 @@
  */
 
 import { HttpServer, ApiServer } from '@mds-core/mds-api-server'
-import { api } from './api'
+import { api, initialize } from './api'
+
+// NOTE: this initialize method has its own try/catch block with error logging
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+initialize()
```

Comment:
Instead of having this as a floating promise, we should wait for it to complete. Either a top-level `await` if it'll work, or `initialize().then(HttpServer(...))`

Reply:
Same goes for the other APIs that do a similar thing

Reply:
i don't think the modules are configured for that, i'll look.